frontend/public/components/_quota.scss

```diff
@@ -21,41 +16,16 @@
   padding-left: 34px;
 }
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.co-quota-dashboard-scopes {
```

Comment:
Sorry, @dtaylor113.  It occurs to me this classname should also include `resource` (e.g., `.co-quota-resource-dashboard-scopes`) and be alphabetized within the file to improve consistency (move rule to line 7).

Reply:
no prob., fixed, thanks :-)